frontend/src/layout/header/UserBox.tsx

```diff
@@ -37,7 +37,12 @@ export const UserBox: React.FC = () => {
     };
 
     if (user === "unknown") {
-        return null;
+        return <Spinner css={{
```

Comment:
Somehow the spinner looks like it's sitting a bit too low for me. :thinking:

Reply:
Then let's ignore it for now. People will only see it very briefly anyway.